package.json

```diff
@@ -28,5 +49,6 @@
     "react-dom": "^15.3.0",
     "react-relay": "^0.9.2",
     "sync-request": "^3.0.1"
-  }
+  },
+  "license" : "MIT"
```

Comment:
I think this should be a link to our license file https://github.com/iporaitech/pwr2-docker/blob/master/LICENSE ... Please correct me if I'm wrong.